localstack-core/localstack/aws/api/s3/__init__.py

```diff
@@ -684,6 +684,12 @@ class InvalidBucketName(ServiceException):
     BucketName: Optional[BucketName]
 
 
+class InvalidBucketOwnerAWSAccountID(ServiceException):
+    code: str = "InvalidBucketOwnerAWSAccountID"
+    sender_fault: bool = False
+    status_code: int = 400
+
+
```

Comment:
This is a bit of a trick, but this file is auto-generated based on the AWS specs, see https://github.com/localstack/localstack/tree/master/docs/localstack-concepts#scaffold---generating-aws-api-stubs

If you'd like create an exception yourself, you can do so in `localstack/services/s3/exceptions.py`, there a few examples you can look into.